test/e2e/authorizer/authorizer_test.go

```diff
@@ -190,6 +189,36 @@ func TestAuthorizer(t *testing.T) {
 				return true
 			}, wait.ForeverTestTimeout, time.Millisecond*100, "User-3 should now be able to list Namespaces")
 		},
+		"without org access, a deep SAR with user-1 against org2 succeeds even with org access for user-1": func(t *testing.T) {
+			t.Log("try to list ConfigMap as user-1 in org without access, should fail")
+			_, err := user1KubeClusterClient.CoreV1().ConfigMaps("default").List(logicalcluster.WithCluster(ctx, org2.Join("workspace1")), metav1.ListOptions{})
+			require.Error(t, err, "user-1 should not be able to list configmaps in org2")
+
+			sar := &authorizationv1.SubjectAccessReview{
+				Spec: authorizationv1.SubjectAccessReviewSpec{
+					ResourceAttributes: &authorizationv1.ResourceAttributes{Namespace: "default", Verb: "list", Version: "v1", Resource: "configmaps"},
+					User:               "user-1",
+					Groups:             []string{"team-1"},
+				},
+			}
+
+			t.Log("ask with normal SAR that user-1 cannot access org2 because it has no access")
```

Comment:
nit: can we please be explicit about the workspace we're asking about? e.g. here we're querying org2:workspace1 (and org2 is some alpha soup ..)

Reply:
fixed